capstone-app/src/app/modules/history/wiki-results.service.ts

```diff
@@ -1,8 +1,19 @@
 import { Injectable } from '@angular/core';
+import { WikiSearchResult } from './WikiSearchTemplate';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
+
 export class WikiResultsService {
```

Comment:
This service needs a test as well. Something like:

```
it('sends an API request to the Wikipedia API', () => {
...
}
```

Basically it should test that the request that actually gets made matches an expected request.

Reply:
I may need help understanding Angular testing to write these. I will do some research on this, it was something I am still a little confused about that I was hoping to get some clarification on in our testing meeting on Friday. Could I add a TODO for the tests until then, and also so I can merge this first? And do the tests on another pr?

Reply:
I think it's worth it to wait for the presentation before submitting. Let me know if you need additional help with this.

Reply:
Ahh, I see. I will try to work on this tonight some.

Reply:
This article might be helpful for testing services specifically: https://medium.com/@shashankvivek.7/testing-services-in-angular-karma-ed49f8d5b264